### ISSUE TITLE
[ArrayTableWidget] add colormap for cell background

### ISSUE THREAD

**Issue by PiRK** [silx-kit/silx#382]
It would be nice to have an option to add a color to the cells' background in the table view of ArrayWidget, based on the data value. (see #368 )

**Comment by vasole** [comment]
It would be nice, but probably  @vallsv idea of having a widget able to toggle image, table and even 3D stack view of the data is simpler and more powerful.

**Comment by PiRK** [comment]
I agree that it would be much nicer to toggle between image and table than to color the cells of the table. 

Or else, to make colored cells useful, one would have to implement some kind of "table zoom" mechanism, to reduce the size of the cells with a slider for example. 

The alternative solution, of displaying the values on the image when enough zoom is applied, is more flexible and maybe easier to implement (?).

**Comment by vasole** [comment]
Yes, the alternative of displaying the values on the image (instead of the image under the values) seems better. At this point I would just keep that possibility in mind.

I would not devote much time to this issue. A simple implementation could be to allow to send an RGB or RGBA array that can be interpreted as numperical_data_set_dimensions x 3 (RGB) or x4 (RGBA). Probably that could be implemented in one day.
